### PR TITLE
Add instances

### PR DESCRIPTION
**Pull request by genofire**
implement #70 
I did not know a nice multilingual solution without make changes of hugo gitea themen.
(is there any reason why it is not inside this repository)

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -38,25 +38,29 @@ menu:
...
       weight: 30
       pre: rss
+    - name: Instances
+      url: /instances
+      weight: 40
+      pre: list
     - name: Code
       url: https://code.gitea.io/
-      weight: 40
+      weight: 50
```
Comment (techknowlogick) [inline config.yaml]: There is no need to modify the weight of the other items in the list. You can set instances weight to 35 and achieve the same outcome.

**Review by techknowlogick — CHANGES_REQUESTED** [review]

File: content/instances/instances.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+toc: true
+draft: false
+url: "instances"
+type: "page"
+---
+
+| Instance-Name                                     | TLS/SSH | IPv6/IPv4 | Free User (public/private) | Free Orgs (public/private) | Limits or Rules |
+|---------------------------------------------------|---------|-----------|----------------------------|----------------------------|--------------|
+| [try.gitea.io](https://try.gitea.io)              |  ✔ / ✔  |   ✔ / ✔   |                ✔ / ✔   |                    ✔ / ✔   | temporary/delete by version-upgrade |
+| [git.spacegit.xyz](https://git.spacegit.xyz)      |  ✔ / ✔  |   ✔ / ✔   |                ✔ / ✘   |                    ✘ / ✘   | [see](https://git.spacegit.xyz/)   |
```
Comment (techknowlogick) [inline content/instances/instances.md]: git.spacegit.xyz offers free private hosting as well, and orgs. (orgs by request). However if users want us to host a different install for them then we charge.

**Comment by lunny** [comment]
And the service is permanent or temporary?

File: content/instances/instances.md (hunk 2 above)
Comment (genofire) [reply content/instances/instances.md]: sry, i miss understand

**Comment by genofire** [comment]
@lunny do you want a guarante column with x years or months?

**Comment by lunny** [comment]
@genofire yes

**Comment by techknowlogick** [comment]
@genofire with teahub coming soon I'm actually going to shutdown git.spacegit.xyz so you can remove it from the list.

**Comment by genofire** [comment]
We run gitlab ATM, sry ...

Be free to hijack this PR

**Pull request closed without merging**